packaging/datadog-agent/source/install_agent.sh

```diff
@@ -40,6 +40,12 @@ if [ -n "$DD_API_KEY" ]; then
     apikey=$DD_API_KEY
 fi
 
+if [ -n "$NO_START" ]; then
```

Comment:
You should probably prefix it with `DD_`, just in case.
It's not really a no_start, since if it's already configured, it's automatically started after the install of the package, but I don't find anything better.